src/eval-frame/components/eval-container.jsx

```diff
@@ -14,9 +15,31 @@ export class EvalContainerUnconnected extends React.Component {
   static propTypes = {
     reportOnly: PropTypes.bool.isRequired,
   }
+  state = {
+    hasError: false,
+    error: '',
+    errorInfo: '',
+  }
+  componentDidCatch(error, info) {
+    this.setState({
```

Comment:
For something like an iframe error, we'd want to bubble that up through the main Redux store, not keep it in local state. Please read up on the Redux paradigm a bit -- in general, using react setState is to be avoided.

Reply:
ok thanks for review